lib/grell/rawpage.rb

```diff
@@ -36,6 +35,17 @@ def has_selector?(selector)
       page.has_selector?(selector)
     end
 
+    def finished_all_ajax_requests?(timeout, interval)
+      Timeout::timeout(timeout) do
```

Comment:
This seems to be similar to what `lib/grell/reader.rb` does. Couldn't we only evaluate jquery without bothering with timing here?